hadoop-hdds/server-scm/src/test/java/org/apache/hadoop/hdds/scm/container/replication/ReplicationTestUtil.java

```diff
@@ -160,6 +160,7 @@ public static ContainerReplica createContainerReplica(ContainerID containerID,
     builder.setDatanodeDetails(datanodeDetails);
     builder.setSequenceId(0);
     builder.setOriginNodeId(originNodeId);
+    builder.setEmpty(keyCount == 0);
```

Comment:
Should this be keyCount or blockCount ?

Reply:
This is in server-scm which doesn't use block count. Here existing test case is just to determine whether replica has some keys or not. If some keys are present then setEmpty as false.